frontend/website/.prettierignore

```diff
@@ -8,6 +8,7 @@ node_modules
 !.env.example
 src/gql
 schema.graphql
+*.svg
```

Comment:
Rather dont ignore it.
Remove this line and then do `mask format` in the root directory.